var/spack/repos/viz/packages/brion/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Brion(CMakePackage):
+    """Blue Brain C++ File IO Library"""
+
+    homepage = "https://github.com/BlueBrain/Brion"
+    git = "https://github.com/BlueBrain/Brion.git"
+    generator = 'Ninja'
+
+    version('develop', submodules=True)
+    version('3.0.0pre', commit='ba34764bbfbba57d0b1e9fbdfc91d3d6785c5225',
+            submodules=True, preferred=True)
+
+    variant('python', default=False, description='Build Python3 wrapping')
+
+    depends_on('ninja', type='build')
+    depends_on('python@3.6.5', type=('build', 'run'), when='+python')
```

Comment:
`python@3.6.5:` to indicate python >= 3.6.5

Reply:
You mean that I have to add a colon at the end?

Reply:
yeah...`:` represent the range, see [here](https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).